packages/design-system/adr/process-ds-transformation.md

```diff
@@ -0,0 +1,87 @@
+# Transform Design system process
+
+## Current status of DS?
+
+- A dedicated team → 2 DS dev + 1 DS designer
+- In _collaboration_ with products designers/content designers
+- _Lack of collaboration_ from product developers
+- _Low usage_ in products
+- How can we move from the DS team to something?
+
+## Org proposals
+
+- Have a composite team from the different domains (1 or 2 devs per domain) and Designers => `Design System Champions 🎖`
+
+  - Goal : Bring the product teams inside the process of build the DS
+
+  - Existing teams (nb of frontend dev):
+
+    - API : Tester / Designer / Creator (1,5?)
+    - DGD : Data Preparation / Data Stewardship / Data Inventory (4,5)
+    - DPD : Pipeline Designer (3)
+    - TDOPS : TMC / Portal / IDAAS (7)
+    - STITCH ?
+
+  - Timeslot Placeholder DS in domains capacity _(see TBD section)_
+    - Jira:
+      - "Main DS ticket" would stay on `TUx`
+      - Tickets would be created in the JIRA's products that take them and be linked to the TUx
+
+## How to construct a roadmap
+
+Discuss the roadmap:
```

Comment:
```suggestion
Discuss the backlog:
```

I've had some time to think about that and design systems do not have roadmaps, in my opinion. Its roadmap is the overall product team's roadmap. 

What it does have is a backlog of tasks that help people fulfil their roadmaps. 

Ours had the following epics: 

- Build foundations with tokens (Done)
- Automate icons and illustration delivery (Done for icons)
- Build all atomic components (closer to done now that I've finally merged the Form updates, but a ways to go still)
- Build better listing patterns (lots of design done, no specs, no code)
- Provide themeing (70% done with the tokens, the 30% left would be implementing logic in ThemeProvider)
- Provide new Dashboard components / layouts and ruleset (some exploration done by Dustin and Maxime)
- Provide new dataviz library (or update the current one) with tokens and dashboard in mind (not even a little bit started).

Reply:
But you'll have some product people who will ask you for a roadmap. As if having one would help anybody.